lib/browser.js

```diff
@@ -196,8 +196,10 @@ PaymentProtocol.verifyCertChain = function(chain, sigHashAlg) {
 
     // Verify current Certificate signature
     if (sigHashAlg !== 'none') {
+      var sigAlgId = c.signatureAlgorithm.algorithm.join('.')
+      var sigAlg = PaymentProtocol.getAlgorithm(sigAlgId).split('_')
       var jsrsaSig = new KJUR.crypto.Signature({
-        alg: sigHashAlg + 'withRSA',
+        alg: sigAlg[1].toLowerCase() + 'withRSA',
```

Comment:
getAlgorithm can return 'RSA'. So the split will return `'1.2.840.113549.1.1.1': 'RSA'` so sigAlg[1] will fail.
https://github.com/bitpay/bitcore-payment-protocol/blob/dc7e7fcdab8e06f212c820f47d6961dd85ec1265/lib/common.js#L48
Can you make it safe?

Reply:
Alright, done and successfully tested in a new browser build of the Bitcoin.com wallet using RSA-SHA256 and RSA-SHA384 certs in the wild.